docs/LoongArch-SMBIOS-Additional-Information-EN.adoc

```diff
@@ -0,0 +1,68 @@
+= LoongArch Processor Additional Information
```

Comment:
Change to be more like the [riscv-smbios](https://github.com/riscv/riscv-smbios) project:

```suggestion
= LoongArch Processor SMBIOS Spec
```